tests/system/CommonFunctionsTest.php

```diff
@@ -486,19 +486,20 @@ public function testIsCli()
 
     public function testDWithCSP()
     {
+        $this->resetServices();
+        $this->resetFactories();
```

Comment:
Factories are already reset by default during `CIUnitTestCase::setUp()` via `$setUpMethods`.

Reply:
This seems to be my mistake. It is not needed.

Removed.